docs/docs/20-usage/20-pipeline-syntax.md

```diff
@@ -444,7 +444,7 @@ when:
 
 #### `evaluate`
 
-Execute a step only if the provided evaluate expression is equal to true. Each [`CI_` variable](./50-environment.md#built-in-environment-variables) can be used inside the expression.
+Execute a step only if the provided evaluate expression is equal to true. Both built-in [`CI_`](./50-environment.md#built-in-environment-variables) and custom variables can be used inside the expression.
```

Comment:
will built-in overwrite custom or other way around ... define clear

Reply:
[Copied](https://github.com/woodpecker-ci/woodpecker/pull/1907/commits/1b9007af4aa5c6b17c5b2066ea1edd3e62840c68) from [Global environment variables](https://github.com/woodpecker-ci/woodpecker/blob/1b9007af4aa5c6b17c5b2066ea1edd3e62840c68/docs/docs/20-usage/50-environment.md?plain=1#L135).  [Implemented](https://github.com/woodpecker-ci/woodpecker/blob/1b9007af4aa5c6b17c5b2066ea1edd3e62840c68/pipeline/frontend/yaml/constraint/constraint.go#L174) according docs.